crates/api_crud/src/post/update.rs

```diff
@@ -85,15 +81,6 @@ impl PerformCrud for EditPost {
       .map(|u| (Some(u.title), Some(u.description), Some(u.embed_video_url)))
       .unwrap_or_default();
 
-    let language_id = Some(
-      data.language_id.unwrap_or(
-        blocking(context.pool(), move |conn| {
-          Language::read_undetermined(conn)
-        })
-        .await??,
-      ),
-    );
-
```

Comment:
My bad.

Reply:
No problem, needs some testing anyway as we dont know how it will work in practice.